doc/source/ray-dashboard.rst

```diff
@@ -0,0 +1,157 @@
+Ray Dashboard
+==============
+Welcome to the Ray dashboard page! 
+
+Ray's built-in dashboard provides metrics, charts, and other features that help Ray users to understand Ray clusters and libraries.
+This includes cluster node level metrics, actor level metrics, configurations, profiling & killing actor buttons, and Ray library metrics such as Tune metrics. 
+Check out the documentation to understand a rich set of features we offer.
+
+Getting Started
+---------------
+  
+Access Dashboard
+~~~~~~~~~~~~~~~~
+If you start Ray on your laptop, dashboard is running by default, and you can access it through its URL. 
+By default the URL is `localhost:8265`. Note that the port will be changed if the default port is not available.
+
+The URL is printed when Ray starts up.
+
+.. code-block:: text
+
+  INFO resource_spec.py:212 -- Starting Ray with 4.2 GiB memory available for workers and up to 2.12 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
+  INFO services.py:1093 -- View the Ray dashboard at localhost:8265
+
+Dashboard is also available when using autoscaler. Check out `Monitoring Autoscaler Status <autoscaling.html#monitoring-cluster-status>`_ for more details.
+
+Example Usage
+-------------
+
+To understand each components, look at `Dashboard Components <ray-dashboard.html#components>`_
+
+Debugging a Blocked Actor
+~~~~~~~~~~~~~~~~~~~~~~~~~
+If creating an actor requires resources (e.g., CPUs, GPUs, or other custom resources) that are not currently available, the actor cannot be created until those resources are added to the cluster or become available.
+This can cause an application to hang. To alert you to this issue, infeasible tasks are shown in red in the dashboard, and pending tasks are shown in yellow.
+You can find this information from the `Logical View` tab. 
+
+- **Infeasible Actor Creation** happens when an actor requires more resources than a Ray cluster can provide.
+- **Pending Actor Creation** happens when there is no available resource for this actor because it is already taken by other tasks / actors.
+
+.. code-block:: python
+  
+  ray.init(num_gpus=2)
+
+  @ray.remote(num_gpus=1)
+  class Actor1:
+      def __init__(self):
+          pass
+  
+  @ray.remote
+  class Actor2:
+      def __init__(self):
+          self.actor1_list = [Actor1.remote() for _ in range(4)]
+
+  @ray.remote(num_gpus=4)
+  class Actor3:
+      def __init__(self):
+          pass
+
+  actor2 = Actor2.remote()
+  actor3 = Actor3.remote()
+
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/dashboard-pending-infeasible-actors.png
+    :align: center
+
+This cluster has 2 GPUs. Actor1 requires 1 GPU to be created. Actor2 creates 4 Actor1. As a result of calling `Actor2.remote()`, 2 Actor1 creation tasks became pending. 
```

Comment:
```suggestion
This cluster has two GPUs, and so it only has room to create two copies of ``Actor1``.
```

Reply:
I will use this. 

This cluster has two GPUs, and so it only has room to create two copies of ``Actor1``. As a result, the rest of Actor1 will be pending.